src/Symfony/Component/HttpFoundation/Tests/BinaryFileResponseTest.php

```diff
@@ -45,6 +45,18 @@ public function testConstructWithNonAsciiFilename()
         $this->assertSame('fööö.html', $response->getFile()->getFilename());
     }
 
+    public function testConstructWithSplTempFileObject()
+    {
+        $file = new \SplTempFileObject();
+        $file->fwrite('foo');
+        $response = new BinaryFileResponse($file);
+
+        $this->assertInstanceOf('Symfony\Component\HttpFoundation\BinaryFileResponse', $response);
+        $this->expectOutputString('foo');
```

Comment:
This is currently failing because `sendContent()` copies zero bytes with `stream_copy_to_stream()`. I would be happy if someone could provide a hint what could be the cause for it.